services/src/main/java/org/jboss/windup/web/fileservlet/FileServlet.java

```diff
@@ -0,0 +1,491 @@
+package org.jboss.windup.web.fileservlet;
+
+/*
+ * Copyright (C) 2009 BalusC
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation, either version 3
```

Comment:
Our project is EPL licensed at the moment. Can we include LGPL code like this in our source?